static/js/components/dashboard/CoursePrice.js

```diff
@@ -1,21 +1,69 @@
 // @flow
 import React from 'react';
 import _ from 'lodash';
+import ReactTooltip from 'react-tooltip';
+import IconButton from 'react-mdl/lib/IconButton';
+import moment from 'moment';
 
-import type { Course } from '../../flow/programTypes';
+import type { Course, CourseRun } from '../../flow/programTypes';
 import {
+  STATUS_NOT_OFFERED,
+  STATUS_PASSED,
+  STATUS_NOT_PASSED,
   STATUS_ENROLLED_NOT_VERIFIED,
+  STATUS_VERIFIED_NOT_COMPLETED,
   STATUS_OFFERED_NOT_ENROLLED,
+  DASHBOARD_FORMAT,
 } from '../../constants';
 import { formatPrice } from '../../util/util';
 
 export default class CoursePrice extends React.Component {
   props: {
     course: Course,
+    now: moment$Moment,
   };
 
+  help(course: Course, now: moment$Moment, firstRun: CourseRun): React$Element<*> {
+    let text = "";
+
+    switch (course.status) {
+    case STATUS_PASSED:
+      text = "Complete!";
+      break;
+    case STATUS_ENROLLED_NOT_VERIFIED:
+      text = "You need to upgrade to the Verified course to get MicroMasters credit";
+      break;
+    case STATUS_NOT_OFFERED:
+      if (firstRun.status === STATUS_NOT_PASSED) {
+        text = 'You failed this course';
+      }
+      break;
+    case STATUS_VERIFIED_NOT_COMPLETED:
+      if (firstRun.course_start_date) {
+        let courseStartDate = moment(firstRun.course_start_date);
+        if (courseStartDate.isAfter(now, 'day')) {
+          let formattedDate = courseStartDate.format(DASHBOARD_FORMAT);
+          text = `Begins ${formattedDate}`;
+        }
+      }
+      break;
+    }
```

Comment:
I agree with @gsidebo -- this would give use an easy way for the tooltip to be conditional on whether the program offer personalized pricing (see #940)